test/util/AclTest.scala

```diff
@@ -0,0 +1,48 @@
+package util
+
+import org.scalatest.{DoNotDiscover, FreeSpec, Matchers}
+import permissions.{TrainingPermission, EmailPermission, EditorialPermission, CommercialPermission}
+
+@DoNotDiscover class AclTest extends FreeSpec with Matchers {
+
+  "Grant access according to collection priorities" - {
+
+    "Allow access to commercial fronts if commercial permissions" in {
+
+      PermissionsChecker.check(AccessGranted, AccessDenied, AccessDenied, Set(CommercialPermission)) should be(AccessGranted)
```

Comment:
It'll get a bit verbose but I think it's worth naming parameters in each test to make them easier to follow.

👍 for the tests, great stuff